openapi-python-templates/requirements.mustache

```diff
@@ -0,0 +1,4 @@
+pydantic >= 1.0
+httpx >= 0.10
```

Comment:
I just created another PR to constrain this dependency as the latest 0.12 version has some backward incompatible changes with the way imports should be done.